src/index.js

```diff
@@ -14,8 +14,8 @@ const store = createStore(
   rootReducer,
   compose(
     applyMiddleware(thunk),
-    // hooks up app to Redux Tools chrome extention
-    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
+    // hooks up app to Redux Tools chrome extension only if available
+    (window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()) || compose
```

Comment:
Previously, if you didn't have the redux devtools installed this would break the app. Adding the `|| compose` prevents it for breaking with people that don't have the redux devtools installed.. which most people viewing your site wont.

Reply:
Good to know! thanks!